.github/workflows/issue-label-automation.yml

```diff
@@ -0,0 +1,50 @@
+name: issue-label-automation
+on:
+  pull_request_target:
+    types: [opened]
+  issues:
+    types: [opened]
+jobs:
+  automate-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    env:
+      IS_BOT: ${{ (github.actor == 'dependabot[bot]' || github.actor == 'mergify[bot]') }}
+      IS_HUMAN: ${{ !(github.actor == 'dependabot[bot]' || github.actor == 'mergify[bot]') }}
+    steps:
+      - name: Check if issue or PR was created by external contributor
+        if: env.IS_HUMAN == 'true' && github.repository_owner == 'celestiaorg'
+        uses: tspascoal/get-user-teams-membership@v3
+        id: teamCheck
+        with:
+          username: ${{ github.actor }}
+          team: "celestia-core"
+          GITHUB_TOKEN: ${{ secrets.PAT_TEAM_CHECK }}
```

Comment:
if we add this, we need an admin on this repo to ensure this secret exists

Reply:
`PAT_TEAM_CHECK` is an organization wide level secret so I don't think I need to add it right?

Reply:
Agreed if it's organization wide.